src/Service/Services/MetadataProviders/SqlMetadataProvider.cs

```diff
@@ -211,15 +207,18 @@ public string GetEntityName(string graphQLType)
                 return graphQLType;
             }
 
-            if (!_graphQLSingularTypeToEntityNameMap.TryGetValue(graphQLType, out string? entityName))
+            foreach ((string entityName, Entity entity) in _entities)
```

Comment:
mentioned earlier, the GraphQLTypeNameToEntityName dictionary enabled quick lookups. possible to preserve?

Reply:
I deliberately tried to remove those members from types because it introduces ways that you can get provider state out of sync with config state, as providers (or other services/etc.) are creating shallow copies.